net-news/sfeed/sfeed-1.8.ebuild

```diff
@@ -0,0 +1,78 @@
+# Copyright 2022-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit toolchain-funcs savedconfig optfeature
+
+DESCRIPTION="Simple RSS and Atom parser"
+HOMEPAGE="https://codemadness.org/sfeed-simple-feed-parser.html"
+SRC_URI="https://codemadness.org/releases/${PN}/${P}.tar.gz"
+
+LICENSE="ISC"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~hppa ~riscv ~sparc ~x86"
+
+THEMES=( mono{,-highlight} newsboat templeos )
+IUSE="+ncurses +${THEMES[@]/#/theme-}"
+REQUIRED_USE="ncurses? ( ^^ ( "${THEMES[@]/#/theme-}" ) )"
+
+DEPEND="ncurses? ( sys-libs/ncurses:= )"
+RDEPEND="${DEPEND}"
+BDEPEND="ncurses? ( virtual/pkgconfig )"
+
+src_configure() {
+	local name
+	for name in "${THEMES[@]}"; do
+		use "theme-${name}" && SFEED_THEME="${name//-/_}"
+	done
+
+	restore_config $(printf "themes/%s.h " "${THEMES[@]//-/_}")
+}
+
+src_compile() {
+	local ncurses_ldflags=""
+	use ncurses && ncurses_ldflags="$($(tc-getPKG_CONFIG) --libs ncurses || die)"
+
+	emake \
+		CC="$(tc-getCC)" \
+		RANLIB="$(tc-getRANLIB)" \
+		AR="$(tc-getAR)" \
+		SFEED_CURSES="$(usex ncurses "sfeed_curses" "")" \
+		SFEED_THEME="${SFEED_THEME}" \
+		SFEED_CURSES_LDFLAGS="${LDFLAGS} ${ncurses_ldflags}"
+}
+
+src_install() {
+	emake \
+		DESTDIR="${D}" \
+		PREFIX="${EPREFIX}/usr" \
+		MANPREFIX="${EPREFIX}/usr/share/man" \
+		DOCPREFIX="${EPREFIX}/usr/share/doc/${PF}" \
+		SFEED_CURSES="$(usex ncurses "sfeed_curses" "")" \
```

Comment:
```suggestion
		SFEED_CURSES="$(usev ncurses sfeed_curses)" \
```

Reply:
Never seen `usev` before - and according to the devmanual, it doesn't seem to take any optional/3rd argument: https://devmanual.gentoo.org/function-reference/query-functions/

Is this some undocumented feature?

Reply:
I see, thanks. I'll update the ebuild in a bit.